main.tf

```diff
@@ -1,18 +1,7 @@
-module "label" {
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.15.0"
-  namespace  = var.namespace
-  name       = var.name
-  stage      = var.stage
-  delimiter  = var.delimiter
-  attributes = var.attributes
-  tags       = var.tags
-  enabled    = var.enabled
-}
-
-resource "aws_launch_template" "default" {
+resource "aws_launch_template" "this" {
   count = var.enabled ? 1 : 0
 
-  name_prefix = format("%s%s", module.label.id, var.delimiter)
+  name = var.name
```

Comment:
If we don't like CloudPosse's label system, can we leave it place but add a way to override it like the following?

```
name_prefix = format("%s%s", coalesce(var.override_name_prefix, module.label.id), var.delimiter)
```

https://www.terraform.io/docs/configuration/functions/coalesce.html

Reply:
Also, if we want to change some of their defaults or have re-usable answers to some of their questions, we can wrap their module in our own.